inc/saiport.h

```diff
@@ -1456,22 +1458,26 @@ typedef sai_status_t (*sai_clear_port_all_stats_fn)(
  *
  * Passed as a parameter into sai_initialize_switch()
  *
+ * @param[in] switch_id Switch id
  * @param[in] count Number of notifications
  * @param[in] data Array of port operational status
  */
 typedef void (*sai_port_state_change_notification_fn)(
```

Comment:
What I mean is not the application side. I mean in the application once we call create with NPU, the adapter could maintain the state (it will anyway need it to do Read, Update and Delete calls for the NPU). So in your way of things how is the OID space, is it per NPU or its a per adapter instance (same OID space for all the NPUs managed by adapter).